alica_ros_turtlesim/README.md

```diff
@@ -124,9 +126,12 @@ In this section, you will create plans using the ALICA plan designer. In this tu
 2. Input Filename `Master` then open the created plan.
 3. Check the masterPlan checkbox in the properties (Icon should turn red).
 4. Add states named `Init` and `Move`
-5. Add transition from `Init` to `Move` and from `Move` to `Init` (Bendpoints can be created by clicking with the active transition tool onto a transition).
+5. Add transition from `Init` to `Move` and from `Move` to `Init` 
+   1. Activate the transition tool, then click on the first state, afterwards on the second state.
+   2. Bendpoints can be created by clicking with the active transition tool onto a transition.
 6. Create an entrypoint (choose default task) and connect it with the `Init` state. 
-8. Create the `Go2RandomPosition` behaviour in the behaviour subfolder .
+7. Create the `Go2RandomPosition` behaviour in the /plans/behaviours folder.
+8. Change the frequency of the ``Go2RandomPosition`` behaviour to 30 Hz in its properties.
```

Comment:
I guess this should only be done for the GoTo behaviour

Reply:
At least my turtle runs to random positions like crazy after I set the frequency of Go2RandomPosition to 30